internal/policies/apparmor/apparmor.go

```diff
@@ -64,8 +64,7 @@ type Manager struct {
 	apparmorParserCmd  []string
 	loadedPoliciesFile string
 
-	muMu    sync.Mutex             // protect pathsMu
-	pathsMu map[string]*sync.Mutex // mutex is per destination path (e.g. machine, users/user1, users/user2)
+	mu *sync.Mutex // Prevents multiple instances of apparmor from running concurrenctly
```

Comment:
I don’t think we need a pointer here, as we only have one manager instance. Let’s keep it simple and avoid using pointers.

Reply:
the pointer is still around? :)